charts/prometheus-rabbitmq-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Rabbitmq metrics exporter for prometheus
 name: prometheus-rabbitmq-exporter
-version: 1.8.0
+version: 1.9.0
```

if this is a fix it should only raise the patch version.

```suggestion
version: 1.8.1
```